src/platforms/kotlin/config.yml

```diff
@@ -0,0 +1,5 @@
+title: Kotlin
+caseStyle: camelCase
+supportLevel: production
+sdk: "sentry.kotlin"
```

Comment:
such SDK does not exist as it falls back to `sentry.java`, should we skip it or set it as `sentry.java`?